### PR TITLE
fix(oidc): grace period for continuity container on oidc callbacks

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: selfservice/strategy/oidc/strategy.go [hunk 1]
```diff
@@ -316,7 +317,10 @@ func (s *Strategy) ValidateCallback(w http.ResponseWriter, r *http.Request) (flo
 
 	cntnr := AuthCodeContainer{}
 	if f.GetType() == flow.TypeBrowser || !hasSessionTokenCode {
-		if _, err := s.d.ContinuityManager().Continue(r.Context(), w, r, sessionName, continuity.WithPayload(&cntnr)); err != nil {
+		if _, err := s.d.ContinuityManager().Continue(r.Context(), w, r, sessionName,
+			continuity.WithPayload(&cntnr),
+			continuity.WithExpireInsteadOfDelete(time.Minute),
```
Comment (hperl) [inline selfservice/strategy/oidc/strategy.go:322]: Are we sure 1 minute is enough?

**Review by hperl — APPROVED** [review]
LGTM!

File: selfservice/strategy/oidc/strategy.go (hunk 1 above)
Comment (aeneasr) [reply selfservice/strategy/oidc/strategy.go:322]: I think it should be enough, at least for the discussed use case

**Comment by aeneasr** [comment]
This was merged as part of another PR

**Comment by aeneasr** [comment]
Or was it?

**Comment by aeneasr** [comment]
It was not.

**Pull request merged**